Regression/Checksum/benchmarks_json/test_2d_particle_EM_diagnostics.json

```
@@ -0,0 +1,25 @@
+{
+  "lev=0": {
+    "Bx": 3864.6903326868182,
+    "By": 239757.36697092294,
+    "Bz": 119878.42982987872,
+    "Ex": 2396689176285.2944,
+    "Ey": 35934018061853.67,
+    "Ez": 71884648325011.86
+  },
+  "electron": {
+    "particle_bx": 3.8909803033142373e-07,
```

```suggestion
    "particle_bx": 3.890980338189904e-07,
```

I would suggest correcting this with what is actually obtained by the github runner (I get the same on my machine, by the way).